Language/Functions/Zero, Due, MKR Family/analogReadResolution.adoc

```diff
@@ -1,7 +1,7 @@
 ---
 title: analogReadResolution()
 categories: [ "Functions" ]
-subCategories: [ "Zero, Due & MKR Family" ]
+subCategories: [ "Zero, Due, MKR family, Nano 33 (BLE and IoT) and Portenta" ]
```

Comment:
This would cause the page to no longer be published at https://www.arduino.cc/reference/en/:

---
![VirtualBox_Ubuntu_20_10_2020_16_58_01](https://user-images.githubusercontent.com/8572152/96657771-e568ba00-12f7-11eb-9ac3-4fa45f0ce37d.png)

---

The reason is because no subcategory of this name has been defined. One could be defined, but I think it's not a sustainable strategy to continuously append board names as needed.

Better would be to change to a non-board specific category name. I would prefer to just move the `analogReadResolution()` and `analogWriteResolution()` reference pages to the "Analog I/O" subcategory, then make sure the reference pages are very clear about which boards support these functions.

What do you think?

Reply:
makes a lot of sense, Per
I don't know how such content is organised, and I'm glad you posted your feedback 👍

Reply:
I have removed the category change and will post it as another PR